similarity/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+
+services:
+    similarity:
+        build: .
+        init: true
+        command: python similarity_server.py
+        volumes:
+            - .:/code
+            - ../freesound-data/similarity_index:/gaia_index
+            - ../freesound-data/analysis
```

Comment:
this file is invalid, it should be fixed

Reply:
it worked in my localhost, but we're no trying to access the directory and that'd fail I guess